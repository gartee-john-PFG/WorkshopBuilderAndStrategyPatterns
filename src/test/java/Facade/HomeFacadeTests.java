package Facade;

import org.approvaltests.Approvals;
import org.junit.jupiter.api.Test;

public class HomeFacadeTests {
    @Test
    public void whatDoesInitialHomeFacadeLookLike(){
        HomeFacade homeFacade = new HomeFacade();
        Approvals.verify(homeFacade.toString());
    }
    @Test
    public void whatDoesPlayMovieOnNetflixLookLike(){
        HomeFacade homeFacade = new HomeFacade();
        homeFacade.playMovieOnNetflix("Star Wars: Episode IV A New Hope");
        Approvals.verify(homeFacade.toString());
    }
}
