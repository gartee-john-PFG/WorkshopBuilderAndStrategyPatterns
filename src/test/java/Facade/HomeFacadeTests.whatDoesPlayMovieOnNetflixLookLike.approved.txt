HomeFacade{
	fan=LivingRoomFan{name='LivingRoomFan \ , status=true \ , currentSpeed=2 },
	stick=LivingRoomFireTV4KStick{name=LivingRoomFireTV4KStick \ status=true \ tv=LivingRoomTV{name='LivingRoomTV \ , status=true \ , currentVolume=1 \ , currentChannel=1 \ , volumeWhenMute=0 \ , source='HDMI ARC } \ appName=Netflix \ contentName=Star Wars: Episode IV A New Hope },
	livingRoomLight=LivingRoomLight{name='LivingRoomLight \ , status=true \ , brightness=20 },
	soundBar=LivingRoomSoundBar{name='LivingRoomSoundBar \ , status=true \ , soundMode='Dolby Atmos \ , currentVolume=20 \ , volumeWhenMute=0 \ , tv=LivingRoomTV{name='LivingRoomTV \ , status=true \ , currentVolume=1 \ , currentChannel=1 \ , volumeWhenMute=0 \ , source='HDMI ARC } },
	tv=LivingRoomTV{name='LivingRoomTV \ , status=true \ , currentVolume=1 \ , currentChannel=1 \ , volumeWhenMute=0 \ , source='HDMI ARC },
	maker=CoffeeMaker{name='CoffeeMaker \ , status=false },
	grill=ElectricGrill{name='ElectricGrill \ , status=false \ , temp=0 },
	kitchenLight=KitchenLight{name='KitchenLight \ , status=false },
	microwave=Microwave{name='Microwave \ , status=false \ , temp=0 \ , time=0 \ , grillOn=false },
	refrigerator=Refrigerator{name='Refrigerator \ , status=false \ , mode='normal },
}